sdk/ruby/lib/ubicloud/model/load_balancer.rb

```diff
@@ -19,6 +19,7 @@ class LoadBalancer < Model
       params[:algorithm] ||= "round_robin"
       params[:stack] ||= "dual"
       params[:health_check_protocol] ||= "http"
+      params[:cert_enabled] ||= false
```

Comment:
This breaks backwards compatibility. We should avoid submitting the parameter if one was not requested by the user, so it uses the default.

```suggestion
```